data/campaigns/tutorial03_seafaring.wmf/scripting/texts.lua

```diff
@@ -154,6 +154,19 @@ expedition2 = {
    )
 }
 
+expedition2a = {
+   position = "topright",
+   title = _"I am going on a trip and I am taking … with me.",
```

Comment:
?

Reply:
well I tried to find an english version of the game "ich packe meinen Koffer und nehme mit ..."

Reply:
If I remember correctly from my earliest childhood English lessons it is called almost as in German "I pack my suitcase and I take…" – but a native speaker should confirm the best phrasing

Reply:
ok I'll do it like in this english wikipedia article then
https://en.wikipedia.org/wiki/I_packed_my_bag